java/ActigraphReader.java

```diff
@@ -443,7 +467,22 @@ private static void readG3TXV1EpochPairs(
     }
 
 
+    public static long getTrueUnixTime(long myTime, String infoTimeShift) {
+        int shiftSign = 1;
+        if (infoTimeShift.charAt(0) == '-') {
+            shiftSign = -1;
+            infoTimeShift = infoTimeShift.substring(1);
+        }
+
+        LocalTime timeShift = LocalTime.parse(infoTimeShift);
+        long timeShiftMilli = 1000 * (shiftSign * timeShift.getHour() * 60 * 60 +
+                timeShift.getMinute() * 60 + timeShift.getMinute()); // time shfit w.r.t. UTC
```

Comment:
Related to your issue discussed in Teams, can you confirm these lines are OK? I'm not at all familiar with this device but it would seem that the minute is added twice?

Reply:
Think you are right. This is a mistake. Thanks for catching this.